apps/web/src/pages/activities/components/ActivityList.tsx

```diff
@@ -33,27 +34,35 @@ export function ActivityList({ data: userData, pagination = false, loading = fal
           return <ActivityItem onClick={onRowClick} key={`activity-item-${item.id}`} item={item} />;
         })}
       </div>
-      {pagination && total > 0 && pageSize > 1 && getPageCount() > 1 && (
-        <Pagination
-          styles={{
-            item: {
-              marginTop: '15px',
-              marginBottom: '15px',
-              backgroundColor: 'transparent',
-              '&[data-active]': {
-                backgroundImage: colors.horizontal,
-                border: 'none',
-              },
-            },
-          }}
-          total={getPageCount()}
-          page={current + 1}
-          onChange={(pageNumber) => {
-            onPageChange(pageNumber - 1);
-          }}
-          position="center"
-        />
-      )}
+
+      <div
+        style={{
+          marginTop: '10px',
+          display: 'flex',
+          width: '100%',
+          justifyContent: 'flex-end',
+          alignItems: 'center',
+          padding: '15px',
+        }}
+      >
+        <Button.Group>
+          <Button
+            variant="default"
+            disabled={pagination?.current === 0 || loading}
+            onClick={() => onPageChange(pagination?.current - 1)}
+          >
+            <FontAwesomeIcon icon="chevron-left" />
```

Comment:
We have Chevron components in the design system:
apps/web/src/design-system/icons/arrows/ChevronLeft.tsx